src/Behat/Behat/Transformation/Transformation/RuntimeTransformation.php

```diff
@@ -8,14 +8,16 @@
  * file that was distributed with this source code.
  */
 
-namespace Behat\Behat\Transformation\Call;
+namespace Behat\Behat\Transformation\Transformation;
 
 use Behat\Behat\Transformation\Transformation;
 use Behat\Testwork\Call\RuntimeCallee;
 
 /**
  * Transformation that is created and executed in the runtime.
  *
+ * @deprecated Will be removed in 4.0. Use specific transformations instead
```

Comment:
btw, if you rename the class, deprecating it is useless: it is already a BC break. you would need to keep it in the ``Call`` namespace